packages/form-builder/package.json

```diff
@@ -0,0 +1,152 @@
+{
+  "name": "@givewp/form-builder",
+  "private": true,
+  "homepage": ".",
+  "scripts": {
+    "start": "node scripts/start.js",
+    "build": "node scripts/build.js",
+    "test": "node scripts/test.js"
```

Comment:
@kjohnson i'm a little concerned with the amount of bundle config setup here. Mainly for maintenance reasons, this would be very hard to keep up with. Is there any reason we can't just use something like Laravel mix or even vite?

A note on workspaces, I think it's a fine place to start with this but we may want to consider turborepo in the future.